test/test_client.py

```diff
@@ -138,6 +138,25 @@ def test_create():
         await test_client.friendbot('bad')
 
 
+@pytest.mark.skip('Our friendbot is configured with 0 starting balance, '
```

Comment:
what's "our" friendbot? the local blockchain one? can we change it?
can we test against testnet instead of a local blockchain?

Reply:
It is the friendbot that I'm using on the local network. And I can change that but it will require changing all other tests since a lot of them assume that friendbot creates accounts with 0 funds.

I will ask Avi to modify the friendbot so I can have this as a separate param

Reply:
👍